CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented (for humans) in this file
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - 2022-03-22
+
+A quick release after realizing that many changes were not released for awhile.
+
+### Added
+- Support for [TACC/launcher](https://github.com/TACC/launcher) use with SLURM and PBS
+### Changed
+- Switched to github actions from travis for CI.
+- Switched to use `datalad push` instead of deprecated `datalad publish`.
+### Fixed
+- Account for git's init.defaultBranch in tests.
+- Invalid escapes in python strings.
+- job_templates: launcher subjob output is now stored in expected files
+
```

Comment:
TODO
- Drop support for Python 3.6 and 3.7
- Add python 3.10
- DNF bugfix https://github.com/ReproNim/reproman/pull/584/commits/cff72856d6ec0c4aa0d9784112ce71aee7c3ca9c
- Drop pip environments older than 20.0.1 [Bump oldest test pip to 20.0.1](https://github.com/ReproNim/reproman/pull/584/commits/8007ec37f7966ed2082ccdea93487a1a0f3e266c)
- Drop `progressbar` support https://github.com/ReproNim/reproman/pull/584/commits/b13e0553be4d7624d68aca329839a8b85f998a22
- Pin paramiko https://github.com/ReproNim/reproman/pull/584/commits/a0b4b055072ba73becc3ef8cc30fa68b2eb68fcd
- Bugfix: keep up with datalad output https://github.com/ReproNim/reproman/pull/584/commits/260ed7363159063cd4856a70a67b25f037d250ef
- Add support for Apptainer v1 (newest release of singularity)
- Change how conda installation URLs are determined https://github.com/ReproNim/reproman/pull/584/commits/11f256ad24f3351ae3562f87331b3a282e8cfb8d